sched/tls/tls_initinfo.c

```diff
@@ -66,5 +70,17 @@ int tls_init_info(FAR struct tcb_s *tcb)
   /* Attach per-task info in group to TLS */
 
   info->tl_task = tcb->group->tg_info;
+
+  /* Initialize cancellation state */
+
+#ifdef CONFIG_CANCELLATION_POINTS
+  /* Set the deferred cancellation type */
+
+  info->tl_cpstate |= CANCEL_FLAG_CANCEL_DEFERRED;
+  info->tl_cpcount  = 0; /* REVISIT: Remove this */
```

Comment:
don't need

Reply:
Can you please clarify what is not needed ? The comment ?

Reply:
I mean the assignment, since info is zerod by default.